graylog2-web-interface/src/components/common/IconButton.tsx

```diff
@@ -49,6 +49,7 @@ type Props = {
   title: string,
   onClick?: () => void,
   className?: string,
+  name: string,
```

Comment:
I know it's not really migrating to TS, but could you also add the `name` prop into the prop types?